setup-tests.ps1

```diff
@@ -0,0 +1,43 @@
+# Check if we're running on Windows, otherwise return as we have nothing to do on other platforms
+if ($IsWindows -eq $false) {
+  return
+}
+
+# Making sure we have the scripts current folder
+$root = $PSScriptRoot
+
+if ("$root" -eq "") {
+  # Fallback for PowerShell 2.0
+  $root = Split-Path -parent "$MyInvocation.MyCommand.Definition"
+}
+
+# Make sure the scripts folder has tests subfolder
+$testPath = Join-Path -Path "$root" -ChildPath "tests"
+
+if (-not (Test-Path -Path "$testPath")) {
+  Write-Error "'tests' folder not found in root`r`n"
```

Comment:
The newlines are not necessary. The entire `Write-Host` family of commands always adds them. Only `Write-Host` itself has a `-NoNewline` option to specifically disable this behavior. `Write-Error` will always add a newline after its message.